frontend/src/api/auth.ts

```diff
@@ -4,6 +4,10 @@ import { components } from "~/types";
 
 import { OidcInfoType } from "./types";
 
+export function ensureCSRF() {
+  return request("GET", "/auth/ensure-csrf");
```

Comment:
If we use the middleware, then we don't need this endpoint anymore, but all endpoint should give the CSRF token